security-admin/src/main/webapp/scripts/views/policies/PermissionList.js

```diff
@@ -701,7 +704,8 @@ define(function(require) {
 			var permList = [];
 			if(this.rangerServiceDefModel.get('name') != XAEnums.ServiceType.SERVICE_TAG.label){
 				if(XAUtil.isAccessPolicy(this.rangerPolicyType)){
-					permList.unshift(localization.tt('lbl.delegatedAdmin'));
+				    // Disable delegating admin privilages
+//					permList.unshift(localization.tt('lbl.delegatedAdmin'));
```

Comment:
ditto. Plus, please make sure that comments are passing style checks for Apache community. Ranger should also have these like other apache hadoop projects.